xarray_sentinel/esa_safe.py

```diff
@@ -57,29 +56,31 @@ def sentinel1_schemas(schema_type: str) -> xmlschema.XMLSchema:
 
 def parse_tag_list(
     xml_path: PathType, schema_type: str, query: str,
-) -> T.List[T.Dict[str, T.Any]]:
+) -> T.Union[T.List[T.Dict[str, T.Any]], T.Dict[str, T.Any]]:
```

Comment:
I knew this was not the correct solution, but I wanted to discuss what to do. In the linked PR you split the function based on the expected output. If it is ready to be merged into main I will merge it back to my branch, otherwise I am open to suggestions.

Reply:
Sorry, I did not realize that you already commented that we need to merge your branch ASAP. Ok for me to switch, then.